package.json

```diff
@@ -47,7 +47,7 @@
   },
   "host": "esdiscuss.org",
   "engines": {
-    "node": "12.3.1"
+    "node": "^12.3.1"
```

Comment:
I'd like to keep this pinned, as it's what heroku uses to determine which version of node to run. Happy for you to bump it to the latest version though.

Reply:
Ah, ok! Taking a look now...

Reply:
For now I'll just bump it to 12.10.0

Reply:
Ah, this doesn't work like this:

```
error esdiscuss.org@5.4.0: The engine "node" is incompatible with this module. Expected version "12.10.0". Got "12.3.1"
```

I'll try with the `x`.